sap_business_document_processing/document_information_extraction_client/constants.py

```diff
@@ -44,4 +66,44 @@
 CONTENT_TYPE_UNKNOWN = 'unknown'
 DATA_TYPE_BUSINESS_ENTITY = "businessEntity"
 DOCUMENT_TYPE_ADVICE = 'paymentAdvice'
+DEFAULT_EXTRACTOR_FIELDS_FILE_PATH = 'default_extractor_fields.py'
 FILE_TYPE_EXCEL = 'Excel'
+
+MODEL_TYPE_DEFAULT = 'defaultWorkflow'
+MODEL_TYPE_LLM = 'llmWorkflow'
+MODEL_TYPE_TEMPLATE = 'templateWorkflow'
+
+SCHEMAS_ENDPOINT = '/schemas'
+SCHEMAS_UPDATE_ENDPOINT = SCHEMAS_ENDPOINT + '/{schemaId}'
+SCHEMAS_LIST_ENDPOINT = SCHEMAS_ENDPOINT + '?clientId={}'
```

Comment:
This constant does not seem to be used anymore, so it can be removed

Reply:
Done